src/main/kotlin/uk/co/craigbass/pratura/math/CurrencyExtensions.kt

```diff
@@ -4,3 +4,5 @@ import java.math.BigDecimal
 
 fun String.toDecimal(): BigDecimal = BigDecimal(this)
 fun Int.toDecimal(): BigDecimal = BigDecimal(this)
+fun BigDecimal?.toCurrencyWithSymbol() = if (this == null) "£0.00" else "£${toCurrencyString()}"
```

Comment:
Paul's suggestion of a Money type seems a lot cleaner than forcing the responsibility of currency formatting into a decimal type